### ISSUE TITLE
How to disable lazyload for images who are already cached ?

### ISSUE THREAD

**Issue by kressly** [eisbehr-/jquery.lazy#213]
If an image has been cached or lazy loaded once, how not to lazy load it again because it is cached ?

**Comment by eisbehr-** [comment]
I don't know a way to do this. You would need to check if a image is in the browser cache, before you render your html to either use lazy or not. But how would you do this?! I would not know how. There are ways to chcek cache with JS, but these checks will load the images completely if not already cached. That said, you will generate an even larger overhead just for the checks, where Lazy loading would be faster.

But what is the problem? Lazy is totally browser-cache compatible and loads images from the cache too. And if you use effects, callbacks or others, Lazy would handle this for you too with cache.

So in short, I don't know how this would be possible and doesn't make sense ...
